### PR TITLE
Develop (challengers js)

### PR DESCRIPTION
**Pull request by doboapps**

### CODE REVIEW

File: staff/doboapps/js/challenges/scripts/create-foreach.test.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+var array = [1, 2, 3];
+
+console.log("forEach: normal use");
+array.forEach(function(elem,i,array){
+    console.log(elem,i,array)   
+});
+
+
+console.log("forEach: function own");
```
Comment (MinaZhen) [inline staff/doboapps/js/challenges/scripts/create-foreach.test.js]: I think that this console.log is useless, you are only displaying a plane text.

File: staff/doboapps/js/challenges/scripts/cube.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+function cube(value) {
+
+    var arrayCube = [];
+
+    if (typeof (value) == 'number') {
+        //console.log(Math.pow(value, 3))
+        return Math.pow(value, 3);
+
+    } else if (typeof (value) == 'object') {
+
```
Comment (MinaZhen) [inline staff/doboapps/js/challenges/scripts/cube.js]: TIP Here you can ask if Array.isArray(value), also you could check in the next for if value[i] is a number.

File: staff/doboapps/js/challenges/scripts/create-foreach.test.js (hunk 1 above)
Comment (doboapps) [reply staff/doboapps/js/challenges/scripts/create-foreach.test.js]: Thanks for the suggestion! 😃  I put the comment for my own compression, in the future 😄

File: staff/doboapps/js/challenges/scripts/cube.js (hunk 2 above)
Comment (doboapps) [reply staff/doboapps/js/challenges/scripts/cube.js]: wow great!!, thank you!

**Pull request merged**